packages/@n8n/permissions/src/roles/scopes/global-scopes.ee.ts

```diff
@@ -141,4 +142,14 @@ export const GLOBAL_MEMBER_SCOPES: Scope[] = [
 	'chatHubAgent:update',
 	'chatHubAgent:delete',
 	'chatHubAgent:list',
+	'apiKey:manage',
+];
+
+export const GLOBAL_CHAT_USER_SCOPES: Scope[] = [
```

Comment:
The chat only user cannot create credentials, so we'd need to
- hide "Configure credentials" menu item in model selector
- change "Please set credentials" callout in ChatPrompt to something else (it's not actionable)

for chat users (or better checking for `credential:create` directly)

Reply:
I see, the empty picker with disabled create option isn't useful but it's not "broken" or exposing unwanted stuff. We can optimize the UX later.